src/main/kotlin/no/nav/familie/ef/sak/arbeidsforhold/ekstern/ArbeidsforholdDomain.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.ef.sak.arbeidsforhold.ekstern
+
+import java.time.LocalDate
+
+class Arbeidsforhold(val navArbeidsforholdId: Long? = null,
```

Comment:
Ser disse finnes i integrasjoner også, burde vi flytte de til kontrakter?

Reply:
Ja, brukte det som utgangspunkt, men behovet for hvilke felter som trengs er ikke avklart. Tenker at det er kjekt om vi kan endre den uten å måtte ha kontrakt på det, da det er forskjellige bruksområder fra app til app (også innad i appen) hvor det er forskjell på hvilke felter som trengs.

Reply:
Ser for meg at det kan bli en del endringer ift det som finnes i integrasjoner etter at behovet er avklart. Mulig det må også endres i integrasjoner også for å få flere felter fra aareg